src/recipes/tooling-engine-singleton.md

```diff
@@ -0,0 +1,119 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Singleton (Anti)-Pattern
```

Comment:
Ok, enough negativity 😛 The admonish block is more than enough to mention the "anti-pattern".

Furthermore, this title should match the summary one: "Engine singletons". With this capitalization.